fluentui_notification/src/main/java/com/microsoft/fluentui/tokenized/notification/Badge.kt

```diff
@@ -72,15 +71,15 @@ fun Badge(
         val textColor = token.textColor(badgeInfo = badgeInfo)
         val typography = token.typography(badgeInfo = badgeInfo)
         val paddingValues = token.padding(badgeInfo = badgeInfo)
+        val shape = RoundedCornerShape(token.cornerRadius(badgeInfo = badgeInfo))
 
         Row(
             modifier
                 .requiredHeight(if (badgeType == BadgeType.Character) 20.dp else 27.dp)
```

Comment:
can we also token height as well.

Reply:
Constraint for height is provided from design. Hence in Figma, there is no token for height as well.